modules/help_editor/ajax/help.php

```diff
@@ -12,9 +12,12 @@
  * @link     https://github.com/aces/Loris
  */
 
+$factory  = \NDB_Factory::singleton();
+$user     = $factory->user();
+$editable = $user->hasPermission('context_help');
+
 try {
-    $factory = \NDB_Factory::singleton();
-    $loris   = new \LORIS\LorisInstance(
+    $loris = new \LORIS\LorisInstance(
```

Comment:
I don't think this is necessary anymore. The `$loris` global is now  defined in AjaxHelper.php.